ci-operator/step-registry/multiarch/tuning-operator/deploy/multiarch-tuning-operator-deploy-commands.sh

```diff
@@ -29,51 +29,54 @@ function createMTOJunit() {
         cat >"${ARTIFACT_DIR}/${filename}.xml" <<EOF
 <?xml version="1.0" encoding="UTF-8"?>
 <testsuite name="${testsuite}" failures="0" errors="0" skipped="0" tests="1" time="1">
-  <testcase name="OCP-00001:lwan:Installing Multiarch Tuning Operator and ClusterPodPlacementConfig should succeed"/>
+  <testcase name="OCP-00001:lwan:Installing Multiarch Tuning Operator should succeed"/>
 </testsuite>
 EOF
     else
         cat >"${ARTIFACT_DIR}/${filename}.xml" <<EOF
 <?xml version="1.0" encoding="UTF-8"?>
 <testsuite name="${testsuite}" failures="1" errors="0" skipped="0" tests="1" time="1">
-  <testcase name="OCP-00001:lwan:Installing Multiarch Tuning Operator and ClusterPodPlacementConfig should succeed">
-    <failure message="">Installing Multiarch Tuning Operator or ClusterPodPlacementConfig failed</failure>
+  <testcase name="OCP-00001:lwan:Installing Multiarch Tuning Operator should succeed">
+    <failure message="">Installing Multiarch Tuning Operator failed</failure>
   </testcase>
 </testsuite>
 EOF
     fi
 }
 
+function wait_created() {
+    # wait 10 mins
+    for _ in $(seq 1 60); do
+        if oc get "${@}" | grep -v -q "No resources found"; then
+            return 0
+        fi
+        sleep 10
+    done
+    return 1
+}
+
 # Deploy Multiarch Tuning Operator
-# Now  we deploy it using the latest bundle images in registry.ci,
-# and will change to via operatorhub when it's ready.
 NAMESPACE=openshift-multiarch-tuning-operator
-oc create namespace ${NAMESPACE}
-OO_BUNDLE=registry.ci.openshift.org/origin/multiarch-tuning-op-bundle:main
-operator-sdk run bundle --timeout=10m --security-context-config restricted -n $NAMESPACE "$OO_BUNDLE"
+if [[ "$MTO_OPERATOR_SOURCE" == "konflux" ]]; then
```

Comment:
```suggestion
if [[ "$MTO_OPERATOR_SOURCE" == "main" ]]; then
```

The catalog in the envs/prow layer is not a konflux one, it always targets the main branch image built by prow.